### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.4 - es. 3.35 (14 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#294] [Esercizi M-Z]
![es.35](https://github.com/user-attachments/assets/761c9e38-da85-4a90-8ecf-a9a438993e07)

**Comment by erinrrr** [comment]
![image](https://github.com/user-attachments/assets/1c4d9f2f-9465-4ae9-afc2-228c70d0a122)

**Comment by marcoseverini** [comment]
<img width="781" alt="immagine" src="https://github.com/user-attachments/assets/4e2a94ee-dba6-4170-89d3-822cb2e988f4">
